Start-C4bCcmSetup.ps1

```diff
@@ -172,21 +172,21 @@ choco pin add --name="'dotnetcore-windowshosting'" --version="'2.2.7'" --reason=
 #Install CCM Web package
 choco install chocolatey-management-web -y --source $PkgSrc --package-parameters-sensitive="'/ConnectionString:Server=Localhost\SQLEXPRESS;Database=ChocolateyManagement;User ID=$DatabaseUser;Password=$DatabaseUserPw;'"
 
-# Completion ntice, and prompt for open of websites
-$Title    = "CCM Setup has now completed."
-$Question = "Would you like to open CCM and Nexus Web Portals in your broswer?"
-$Choices  = '&Yes', '&No'
-
-$decision = $Host.UI.PromptForChoice($Title, $Question, $Choices, 1)
-if ($Decision -eq 0) {
-    start msedge http://localhost/Account/Login,http://localhost:8081/#browse/browse
-    Write-Host 'Script completed. Have a wonderful day!' -ForegroundColor Green
-} else {
-    Write-Host 'Script completed. Have a wonderful day!' -ForegroundColor Green
+$CcmSvcUrl = choco config get centralManagementServiceUrl -r
+$CcmJson = @{
+    CCMServiceURL = $CcmSvcUrl
+    CCMWebPortal = "http://localhost/Account/Login"
+    DefaultUser = "ccmadmin"
+    DefaultPwToBeChanged = "123qwe"
+    CCMDBUser = $DatabaseUser
 }
+$CcmJson | ConvertTo-Json | Out-File .\ccm.json
 
-#Stop logging
-Stop-Transcript
+# Completion notice
```

Comment:
This comment seems unnecessary

Reply:
Good call. We should probably remove these in all the scripts then.